### PR TITLE
Transaction Basics - Chia Academy

### PR DESCRIPTION
**Pull request by braunswag**
A massive thank you to Slowesttimelord for his help!

### CODE REVIEW

File: docs/academy/blockchain-basics/transaction-basics.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+---
+
+## Knowledge check
+
+:::tip Question 1 - Transaction Fees
+
+Who receives the transaction fee?
+
+A. The transaction fee gets burned.
```
Comment (BrandtH22) [inline docs/academy/blockchain-basics/transaction-basics.md:58]: This line will need two spaces added to the end of it to ensure the list items are separated into a list (1/3)

File: docs/academy/blockchain-basics/transaction-basics.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+B. It goes to the farmer that formed the block.
```
Comment (BrandtH22) [inline docs/academy/blockchain-basics/transaction-basics.md:59]: This line will need two spaces added to the end of it to ensure the list items are separated into a list (2/3)

File: docs/academy/blockchain-basics/transaction-basics.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+C. It goes to the receiver of the transaction.
```
Comment (BrandtH22) [inline docs/academy/blockchain-basics/transaction-basics.md:60]: This line will need two spaces added to the end of it to ensure the list items are separated into a list (3/3)

**Review by BrandtH22 — CHANGES_REQUESTED** [review]

File: docs/academy/blockchain-basics/transaction-basics.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+title: Sending Transactions
+slug: /transactions-basics
```
Comment (SlowestTimelord) [inline docs/academy/blockchain-basics/transaction-basics.md:3]: The slug should match the file name as well, removing the `s` from `transactions`

**Comment by BrandtH22** [comment]
Hey Austin, as relayed in DMs we have made note of this content and will be using it in other academy lessons so this pr will be closed as the page will not be merged as is. Let us know if any questions arise and thank you for the contribution!

**Pull request closed without merging**